openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -152,6 +154,12 @@ def test_update_xblocks_cache(self, course_attr, expected_sql_queries):
         """
         course = getattr(self, course_attr)
 
+        if settings.ROOT_URLCONF == 'lms.urls':
+            # When the tests run under LMS, there is a certificates course_published
+            # signal handler that runs and causes the number of queries to be one more
+            # (due to the check for disabled_xblocks in django.py).
+            expected_sql_queries += 1
```

Comment:
All of the unit tests in common/openedx run under both LMS and CMS. This is a long-standing pain point, but perhaps theoretically necessary to ensure that the code works correctly under both environments?